QuadRelate.Tests/BoardHelperTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Moq;
+using QuadRelate.Helpers;
+using QuadRelate.Types;
+using System.Collections.Generic;
+using Xunit;
+
+namespace QuadRelate.Tests
+{
+    public class BoardHelperTests
+    {
+        private readonly BoardHelper _boardHelper;
+        private readonly Board _board;
+
+        public BoardHelperTests()
+        {
+            _boardHelper = new BoardHelper();
+            _board = new Board();
+        }
+
+        [Fact]
+        public void AvailableColumns_ForFullBoard_ReturnsEmptyList()
+        {
+            // Fill board in with red counters
```

Comment:
Duplicated code.
Add a private method, or even a helper class.
Or possible add a Board.Fill(Cell cell). You'll need to initialize the board anyway.